otter/test/integration/lib.py

```diff
@@ -81,6 +87,118 @@ def record_result(r):
         ).addCallback(record_result)
 
 
+@attributes([
+    Attribute('group_config', instance_of=dict),
+    Attribute('pool', default_value=None),
+])
+class ScalingGroup(object):
+    """This class encapsulates a scaling group resource.  It provides a means
+    which lets you create new scaling groups and, later, automatically
+    dispose of them upon integration test completion.
+    """
+
+    def stop(self, rcs):
+        """Clean up a scaling group.  Although safe to call yourself, you
+        should think twice about it.  Let :method:`start` handle registering
+        this function for you.
+
+        At the present time, this function DOES NOT stop to verify
+        servers are removed.  (This is because I haven't created
+        any tests which create them yet.)
+        """
+
+        return (self.get_scaling_group_state(rcs)
```

Comment:
Non-blocking:  this is totally fine, but destroying a scaling group in autoscale is idempotent (it's a bug if it isn't).
So probably you can just delete it without checking state first, and either a 404 or a 204 are ok?